app/scripts/lib/app-start.js

```diff
@@ -110,6 +111,7 @@ function (
   User,
   FormPrefill,
   Notifications,
+  PageRefreshWatch,
```

Comment:
Actually, while I'm picking out colours for your new bikeshed, I wonder if `Listener` or `Observer` might be more a more appropriate suffix than `Watch`? :grin: